package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "webpack-deploy",
-  "version": "0.13.1",
+  "version": "0.14.0",
```

Comment:
Is there any change in the project itself? any new feature? IMO this should be a patch update.